pkg/cli/common.go

```diff
@@ -161,7 +161,8 @@ func GetBudFlags(flags *BudResults) pflag.FlagSet {
 	fs.BoolVar(&flags.PullAlways, "pull-always", false, "pull the image, even if a version is present")
 	fs.BoolVarP(&flags.Quiet, "quiet", "q", false, "refrain from announcing build instructions and image read/write progress")
 	fs.BoolVar(&flags.Rm, "rm", true, "Remove intermediate containers after a successful build")
-	fs.StringVar(&flags.Runtime, "runtime", util.Runtime(), "`path` to an alternate runtime. Use BUILDAH_RUNTIME environment variable to override.")
+	//	Remove next due to flag name collision in podman build.  Define in cmd/buildah/bud.go.
```

Comment:
Just remove it, do not comment it out.

Reply:
OK, but I'm planning to leave a comment in here to make it easier to find and to avoid the issue going forward.